### PR TITLE
change_blast_tutorial

### PR DESCRIPTION
**Pull request by deming7h777**

### CODE REVIEW

File: blast/templates/blast/manual.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+<p> </p>
+<p><span>When the BLAST program has finished, your results will be displayed. Query and subject coverage graphs are displayed on the top left and right of the results panel, respectively. A tabular view of the results is on the bottom left, and raw text output on the bottom right.</span></p>
+<p><img src={% static 'blast/images/blast_result_page.png' %} style="border:1px groove;width:1200px;height:600px"></p>
+<p> </p>
+<p><span>The feature heights on the query and subject coverage graphs can be modified using the right scroll bar.</span></p>
+<p><img src={% static 'blast/images/blast_result_zoom.png' %} style="border:1px groove;width:1200px;height:600px"></p>
+<p> </p>
+<p><span>Scrolling over individual HSPs in any of the four panels will highlight the HSP across all panels - orange in the query and subject coverage graphs, and yellow in the results table and raw text output.</span></p>
+<p><img src={% static 'blast/images/blast_result_zoom.png' %} style="border:1px groove;width:1200px;height:600px"></p>
+<p><span><img alt="" src="/sites/default/files/blast_result_scroll.png" /></span></p>
```
Comment (hsiaoyi0504) [inline blast/templates/blast/manual.html]: this line will not correctly referencing the **blast_result_scroll.png**.

File: blast/templates/blast/manual.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+{% block content %}
+
+<div class="main-container container">
+
+
+  <div class="row">
+
+    
+    <section class="col-sm-12">
+            <ol class="breadcrumb"><li><a href="/" title="">Tutorials and Resources</a></li>
```
Comment (hsiaoyi0504) [inline blast/templates/blast/manual.html]: It seems to me that this url will return 404 and we should remove it.

**Comment by hsiaoyi0504** [comment]
Another thing is that the manual will be accessed from main page of blast app. We should modify the link as well.

https://github.com/NAL-i5K/genomics-workspace/blob/93a88251d96c40ae14d9ae72fbc7b3d630c07376/blast/templates/blast/main.html#L38

**Comment by deming7h777** [comment]
i have updated the blast tutorial which follow hmmer's format

File: blast/templates/blast/manual.html (hunk 1 above)
Comment (hsiaoyi0504) [reply blast/templates/blast/manual.html]: @deming7h777 Don't forget to fix this part.

File: blast/templates/blast/main.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
 <div class="tripal_organism-info-box tripal-info-box">
 <!--<a href="docs/blast_databases.html">-->
 <fieldset id="fieldset-datasets">
-    <legend>BLAST Databases <a href="https://i5k.nal.usda.gov/content/blast-tutorial" target="_blank"><span id="tutorial"><span class="glyphicon glyphicon-book"></span> Tutorial</span></a></legend>
+    <legend>BLAST Databases <a href="https://i5k.nal.usda.gov/webapp/blast/manual/" target="_blank"><span id="tutorial"><span class="glyphicon glyphicon-book"></span> Tutorial</span></a></legend>
```
Comment (hsiaoyi0504) [inline blast/templates/blast/main.html]: The thing needs to be considered here is that this website project is intended for general purpose. We don't want to hard-coded this. Instead, we want make this link go to the url `<host>/blast`.

**Comment by deming7h777** [comment]
I changed the problem above please take a look

**Pull request merged**